codacy/values.yaml

```diff
@@ -244,6 +244,28 @@ engine:
     grafana_dashboards:
       enabled: false
 
+codacy-ingress:
```

Comment:
Do we need to explicitly include this section in the values.yaml file, with "create: false"? In the documentation we are recommending that customers append the values-production.yaml file to this one, so the codacy-ingress section will be repeated and will probably trigger some conflict.

Reply:
yes. it needs to be here to pass the linting. The values.yaml usually has defaults